src/main/java/org/spongepowered/common/service/permission/SpongePermissionService.java

```diff
@@ -180,4 +188,13 @@ public void addDescription(PermissionDescription permissionDescription) {
     public SpongeSubjectCollection getDefaultCollection() {
         return defaultCollection;
     }
+
+    public void setupDefaultPermissions() {
+        getGroupForOpLevel(SELECTOR_LEVEL).getSubjectData()
+                .setPermission(SubjectData.GLOBAL_CONTEXT, SELECTOR_PERMISSION, Tristate.TRUE);
```

Comment:
I like that you have placed this into SpongeCommon, but I'd sooner see it not a method of the permissions service. 
Sponge should set up it's default permissions similar to how plugins do it, not make it optional based on what permissions service is running.